electronics_lib/Passives.py

```diff
@@ -314,6 +297,195 @@ def derated_capacitance(row: Dict[str, Any]) -> Tuple[float, float]:
       raise ValueError(f"no single capacitors in ({capacitance}) F")
 
 
+class SmtCeramicCapacitorGeneric(Capacitor, FootprintBlock, GeneratorBlock):
+
+  SINGLE_CAP_MAX = 22e-6
+
+  @init_in_parent
+  def __init__(self, *args, **kwargs):
+    super().__init__(*args, **kwargs)
+
+    self.footprint_spec = self.Parameter(StringExpr(""))
+    self.derating_coeff = self.Parameter(FloatExpr(1))
```

Comment:
May be useful to note in a comment here that this is capacitance derating, as a simple multiplier, and does not scale with package or applied voltage.